src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: rgba(0, 0, 0, 0.45) 0 15px 10px -20px;
+}
+
+.logo,
+.logo__link {
+  height: 40px;
+  width: 40px;
+}
+
+.logo {
+  display: flex;
+  align-self: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  margin: 0 10px;
+  position: relative;
+}
+
+.nav__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
```

Comment:
this is too much. I'm pretty sure you can remove this and simply set the line-height equal to the height;

Reply:
No problem I'll fix all of them, thanks!